### DISCUSSION TITLE
How to use Flow-Ja-Face file to draw the flow direction figure

### DISCUSSION THREAD

**Discussion by Sakura223Leslie** [modflowpy/flopy#2262] [Q&A] [answered]
The latest version of flow- ja- face dose not have the flow right/left face. How we can use flow-ja-face file to draw the flow direction figure? Thanks!

**Comment by mwtoews** [comment, marked as answer]
Take a look at [`flopy.mf6.utils.postprocessing.get_structured_faceflows()`](https://flopy.readthedocs.io/en/stable/source/flopy.mf6.utils.postprocessing.html#flopy.mf6.utils.postprocessing.get_structured_faceflows) to get the face flows for the flow right face, flow front face, and flow lower face from the MODFLOW 6 flowja flows. You can see examples how to use it with [test_postprocessing.py](https://github.com/modflowpy/flopy/blob/19ddf6bf3331ab022b55788aca5a7e8471051b41/autotest/test_postprocessing.py), e.g.:
```python
    mf6_cbc = gwf.output.budget()
    mf6_flowja = mf6_cbc.get_data(text="FLOW-JA-FACE")[0]
    mf6_frf, mf6_fff, mf6_flf = get_structured_faceflows(
        mf6_flowja,
        grb_file=mf6_ws / "freyberg.dis.grb",
    )
```
